integration-tests/hibernate-reactive-mysql/pom.xml

```diff
@@ -205,8 +204,8 @@
                                             <date>default</date>
                                             <color>cyan</color>
                                         </log>
-                                        <!--                                         Speed things up a bit by not actually flushing writes to disk -->
-                                        <!--                                         <tmpfs>/var/lib/mysql</tmpfs> -->
+                                        <!-- Speed things up a bit by not actually flushing writes to disk -->
+                                        <tmpfs>/var/lib/mysql</tmpfs>
```

Comment:
FWIW it's uncommented in every other place we have a mysql container, including in the Vert.x MySQL client tests.
So it would be a huge red flag if we needed to comment this out only when testing Hibernate Reactive.